### PR TITLE
 Added server_tokens property (defaults to on)

### PR DESCRIPTION
**Pull request by ls-hector**
Added server_tokens property which defaults to on, in order to make it easier to disable it on the proxy cookbooks and future changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # nginx\_resources cookbook changelog
 
+## v0.8.0
```
Comment (ls-michielrensen) [inline CHANGELOG.md]: v0.9.0 ;-)

**Review by ls-michielrensen — CHANGES_REQUESTED** [review]
Did you try `kitchen converge`?

File: CHANGELOG.md (hunk 1 above)
Comment (ls-hector) [reply CHANGELOG.md]: thanks! changed

**Comment by ls-hector** [comment]
Re-ran kitchen converge

**Review by ls-michielrensen — APPROVED** [review]

**Pull request merged**